test/jupyter-test.el

```diff
@@ -260,6 +260,16 @@
           (jupyter-insert :text/html "<p>hello</p>")
           (should html-parser-called))))))
 
+(ert-deftest jupyter-find-mime-types ()
```

Comment:
Can you place the test with all the other `org` related tests at the bottom of the file. Also can you name the test the same as the function name, `jupyter-org--find-mime-types`. ERT tests are not normal functions so there is no name clashing.